docs/sources/clients/promtail/stages/multiline.md

```diff
@@ -70,7 +70,7 @@ multiline:
 
 ### Custom Log Format
 
-The previous example assumed you had no control over the log format. Thus it required a more ellaborrate regular expression to match the first line. However, if you can control the log format of the system under observation we can simplify the first line matching.
+The previous example assumed you had no control over the log format. Thus it required a more elaborate regular expression to match the first line. However, if you can control the log format of the system under observation we can simplify the first line matching.
```

Comment:
Thank you for the spelling correction. Would you mind correcting the punctuation and wording, since you've gone this far?

```suggestion
The example assumed you had no control over the log format. Thus, it required a more elaborate regular expression to match the first line. If you can control the log format of the system under observation, we can simplify the first line matching.
```

Reply:
@bigfist02, although unrelated to your PR, would you like to take care of committing the suggested changes while we are at it?